gradle.properties

```diff
@@ -1,6 +1,6 @@
-detektIntellijPluginVersion=0.3.0
-detektVersion=1.0.0-RC16
-kotlinVersion=1.3.40
+detektIntellijPluginVersion=1.0.0
```

Comment:
I would't base this version on detekt's version. It's just based on detekt but the way it presents its features is very limited. Bugs like the debugging/vcs exception does not make it 1.0 imo.
I would just go with 0.3.1 and wait if someone has the skills and time to invest in this plugin.
Don't you agree?

Reply:
Yes, you are right. In this case it doesn't make sense.
There a few blockers (bugs) that hinder a good version 1.0.
I'll change it back.